contracts/truefi2/LoanToken2.sol

```diff
@@ -516,7 +513,7 @@ contract LoanToken2 is ILoanToken2, ERC20 {
     }
 
     function version() external override pure returns (uint8) {
-        return 6;
+        return 7;
```

Comment:
The current on chain version is `6`. I don't think we should bump it any higher.

Reply:
It's a very major change. Why wouldn't we bump version number here?